src/main/scala/scalafix/sbt/ScalafixEnable.scala

```diff
@@ -30,7 +33,45 @@ object ScalafixEnable {
     } yield p -> fullVersion
   }
 
-  lazy val command = Command.command(
+  lazy val command =
+    if (isSemanticdbPluginAvailable) withSemanticdbPluginCommand
+    else withoutSemanticdbPluginCommand
+
+  private lazy val withSemanticdbPluginCommand = Command.command(
+    "scalafixEnable",
+    briefHelp = "Configure SemanticdbPlugin and scalaVersion for scalafix.",
+    detail =
+      """1. enables SemanticdbPlugin with the right version supported by scalafix
+        |2. sets scalaVersion to latest Scala version supported by scalafix""".stripMargin
```

Comment:
This changes the implementation of `scalafixEnable` for sbt 1.3+ users, so it has the potential of breaking a few things, especially on complex projetcts with many modules and/or scala versions. I wonder if we could smoke-test it with scala steward that relies on it?

Reply:
Yes, using `SemanticdbPlugin` should be equivalent, but there are subtle changes in terms of scopes that might make it regress (see my comment below also).

Reply:
In the amended commit, I have reworked how SemanticdbPlugin is enabled to make sure it's matching where `semanticdb-scalac` was currently injected AND on scala3.